Core/Core/Inbox/ComposeMessage/ViewModel/ComposeMessageViewModel.swift

```diff
@@ -103,6 +104,8 @@ final class ComposeMessageViewModel: ObservableObject {
     private var hiddenMessage: String = ""
     private var autoTeacherSelect: Bool = false
     private var teacherOnly: Bool = false
+    private var sendIndividualToggleTemp: Bool = false
```

Comment:
Can we have a more telling name here? Like `sendIndividualToggleLastValue`? This variable is definitely not temporary.